drivers/hmis/app/graphql/types/base_audit_event.rb

```diff
@@ -103,9 +104,24 @@ def user
       # This happens when a Job updates records, which we should display as System changes.
       return User.system_user if object.whodunnit == 'unauthenticated'
 
-      # If user was impersonating, return the true user only. If someone performed the action while impersonating, whodunnit stores as '1 as 2' where 1 is the true user.
```

Comment:
It would be helpful to keep some comment explaining what the "x as x" string means